pyramid/file_test.go

```diff
@@ -0,0 +1,134 @@
+package pyramid
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/google/uuid"
+)
+
+func TestPyramidWriteFile(t *testing.T) {
+	filename := uuid.Must(uuid.NewRandom()).String()
+	filepath := path.Join("/tmp", filename)
```

Comment:
This is not a safe place.  Why not just [`ioutil.TempFile`](https://golang.org/pkg/io/ioutil/#TempFile)?